learn/experimental/log_customization.mdx

```diff
@@ -0,0 +1,73 @@
+---
+title: Log customization — Meilisearch documentation
+description: "Customize Meilisearch logs with two experimental features: --experimental-logs-mode and --experimental-enable-logs-route."
+---
+
+# Log customization
+
+Customize Meilisearch logs with two experimental features: `--experimental-logs-mode` and `--experimental-enable-logs-route`.
+
+## Customize log output
+
+Use `--experimental-logs-mode` to customize log output:
+
+```sh
+./meilisearch --experimental-logs-mode json
+```
+
+`--experimental-logs-mode` accepts either `human` or `json`. `human` is the default value.
+
+For more information, [consult this feature's GitHub discussion](https://github.com/orgs/meilisearch/discussions/723).
+
+## Log routes
+
+Activate the `/logs` routes during runtime using the `/experimental-features` route:
+
+```sh
+curl \
+  -X PATCH 'http://localhost:7700/experimental-features/' \
+  -H 'Content-Type: application/json'  \
+  --data-binary '{
+    "logsRoute": true
+  }'
+```
+
+Use `/logs/stderr` to customize logging levels for the default logging system:
+
+```sh
+curl \
+	-X POST http://localhost:7700/logs/stderr \
+	-H 'Content-Type: application/json' \
+	--data-binary '{
+      "target": "milli=trace,index_scheduler=info,actix_web=off"
+  }'
+```
+
+Use the `POST` endpoint of `/logs/stream` and a listener for focused debugging sessions:
+
+```sh
+curl \
+	-X POST http://localhost:7700/logs/stream \
+	-H 'Content-Type: application/json' \
+	--data-binary '{
+    "mode": "human",
+    "target": "index_scheduler=trace"
+  }'
+```
+
+Configure the stream using the same parameters as regular logging: `mode` and `target`. The stream will continue to run indefinitely until you interrupt it.
+
+Use the `DELETE` endpoint of `/logs/stream` to interrupt a stream:
```

Comment:
```suggestion
Configure the stream using the same parameters as regular logging: `mode` and `target`. The stream will continue to run indefinitely until you interrupt it.

<Capsule intent="warning">
Take care while using this feature; your HTTP client might not show you and wait until the meilisearch itself disconnects you.

If you're using `curl`, for example, that won't be an issue, and you can simply `ctrl+c` your request once you've got enough logs.
But for `httpie`, `xh`, or other clients, you must close the stream from Meilisearch with the `DELETE` endpoint below.
</Capsule>

Use the `DELETE` endpoint of `/logs/stream` to interrupt a stream:
```

Hey, I think we should put a big warning here because even I forgot about it twice; Kero had the issue once, and our first open-source user had the issue as well 😅